submissions/idowuMariam/html-forms/Contact-form.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
Before working on `Add page Navigation` task, I believe that you must have work on creating a contact form page before. So, you don't have to create a new contact form page again. Kindly work on adding navigation links to those already existing pages